core/SizeSystem.js

```diff
@@ -113,7 +113,7 @@ function proportionalSizeChanged (node, components, size) {
 }
 
 function differentialSizeChanged (node, components, size) {
-    var differentialSize = size.getDifferentialSize();
```

Comment:
The function in Size was called `getDifferential`. About 50% of the functions were used/ defined as get*Size, the other half was just get*. I changed everything to get*. I don't care which format we choose, but this was broken.